src/logger.py

```diff
@@ -37,40 +42,27 @@ def exception(self, message: str):
         self.logger.flush()
 
 
-def route_logger(logger: Logger):
-    """
-    Decorator factory that creates a decorator to log route entry and exit points.
-    The decorator uses the provided logger to log the information.
```

Comment:
Any reason to leave out these comments? Please consider leaving the comments or expanding them, including the new param `func`, which can have a better name, like `routerFunction` instead

Reply:
Added the comments